### ISSUE TITLE
ToF

### ISSUE THREAD

**Issue by hu-po** [worldcoin/orb-hardware#12]
afaik+fwiw: ToF sensors are sensitive to ambient lighting, can become a huge headache irl.

**Comment by fbodenst** [comment]
That's correct. However, we've only observed critical issues in direct sunlight. Over time, as we required users to expose a sufficiently large portion of their iris, we naturally gravitated towards indoor locations. In such environments, the data quality is good enough for guiding the user. We don't need to conduct measurements demanding high accuracy; we use it mainly to detect if someone is present and/or approximately within the valid working range of the main optical system. Following that, a different method takes charge to direct the gimbal and capture a clear image of the iris.
